src/config.c

```diff
@@ -398,7 +398,7 @@ get_default(settings_t *def)
       def->LOGFILE = strdup(pch);
       rmn(def->LOGFILE);
     }
-    log_path = def->LOGFILE;
+    log_path_conf = def->LOGFILE;
```

Comment:
Well, we use `log_path` in log.c to define PATH of logfile.
You can't just change this name here; `logpath` will be NULL in `history()` func.